src/Hosting/Hosting/src/Http/DefaultHttpContextFactory.cs

```diff
@@ -71,11 +71,15 @@ public void Dispose(HttpContext httpContext)
             }
         }
 
-        internal void Dispose(DefaultHttpContext httpContext)
+        internal void Dispose(DefaultHttpContext httpContext, out bool usedHttpContextAccessor)
```

Comment:
Why is this an out param rather than just a property on the class? The code in this method does not influence the result, _httpContextAccessor always has or does not have a value.

Reply:
True.